.gitignore

```diff
@@ -0,0 +1,64 @@
+# Variable files
```

Comment:
Add the terraform lock, license and contributing files.
Also, a basic pipeline would be great